src/components/AprInfo/AprInfo.tsx

```diff
@@ -86,14 +88,22 @@ export function AprInfo({
             </Trans>
           </div>
         )}
-        {isIncentiveActive && (
-          <div>
-            <Trans>
-              The Bonus APR will be airdropped as {airdropTokenTitle} tokens.{" "}
-              <ExternalLink href={getIncentivesV2Url(chainId)}>Read more</ExternalLink>.
-            </Trans>
-          </div>
-        )}
+        {isIncentiveActive &&
+          (isTbtcIncentive ? (
+            <div>
+              <Trans>
+                The Bonus APR will be airdropped as {airdropTokenTitle} and T tokens.{" "}
+                <ExternalLink href={TBTC_INFORMATION_URL}>Read more</ExternalLink>.
+              </Trans>
+            </div>
+          ) : (
```

Comment:
as I understand we won't support simultaneous incentives programs. so it's either T incentives or ARB?

Reply:
both tokens would be rewarded

<img width="618" alt="image" src="https://github.com/user-attachments/assets/781cec9b-2eaf-4c28-a603-3c40763a7879">
https://x.com/GMX_IO/status/1833896397709463921

Reply:
@gmxer in the end we decided to remove tbtc so only arb is left